Update README.md (#131)
informalsystems/hydro: README.md

# What is Hydro?

> Hydro is a auction platform where ATOM stakers can lock up their staked Atoms in exchange for voting power, and use it to vote on proposals to deploy tokens owned by the community pool into liquidity (aka market-making) positions on various decentralized exchanges. The liquidity itself is owned by the Cosmos Hub (although initially custodied by a committee), thus this is a way of deploying Protocol Owned Liquditiy (PoL). The advantage of PoL compared to issuing incentives (i.e. paying rewards to liquidity providers in certain pools) is that the community pool does not *spend* its funds, but rather deploys them. This means that the funds can be re-used in the future, and the community pool can grow over time. Projects using the Cosmos Hub’s Interchain Security platform receive several beenfits (special tranches, longer export duration etc.)

Reviewer: ```suggestion
Hydro is an auction platform where ATOM stakers can lock up their staked ATOMs in exchange for voting power to vote on proposals for deploying tokens owned by the community pool into liquidity (aka market-making) positions on various decentralized exchanges. 
The liquidity itself is owned by the Cosmos Hub (although initially custodied by a committee), thus this is a way of deploying Protocol Owned Liquditiy (PoL). 
The advantage of PoL compared to issuing incentives (i.e. paying rewards to liquidity providers in certain pools) is that the community pool does not *spend* its funds, but rather deploys them. 
This means that the funds can be re-used in the future, and the community pool can grow over time. 
Projects using the Cosmos Hub’s Interchain Security platform receive several beenfits (special tranches, longer export duration etc.)
```

> Hydro is implemented based on its [litepaper](https://forum.cosmos.network/t/atom-wars-introducing-the-hydro-auction-platform/13842). That said, we've already incorporated and will continue to incorporate feedback, which means that implementation will increasingly diverge from the paper. Up-to-date Hydro documentation will be posted on the Hydro website.

Reviewer: Do we have a link to the Hydro website? Even if it's just a message "coming soon".

# Main features

The main features of the Hydro smart contract are as follows:

## Locking

> Users can lock their Atoms in the Hydro smart contract to receive voting power. Concretely, users can lock liquid staking module shares aka LSM shares in Hydro. Each LSM share represents a staked Atom that is delegated to a certain validator. Learn more about the Liquid Staking Module [here](https://github.com/iqlusioninc/liquidity-staking-module). This means users that currently have liquid Atom tokens first need to stake their tokens before they can lock them in Hydro. There are different locking durations available, and the longer the duration, the more voting power the user receives. Once the locking period is over, the user can unlock their locked tokens and will receive the original amount of tokens back. It is also possible to refresh a partially expired lockup to gain more voting power (read more about *Voting* below).

Reviewer: ```suggestion
Users can lock their ATOMs in the Hydro smart contract to receive voting power. 
Concretely, users can lock Liquid Staking Module (LSM) shares in Hydro. 
Each LSM share represents a staked ATOM that is delegated to a certain validator. 
Learn more about the LSM [here](https://hub.cosmos.network/main/modules/lsm-staking). 
This means users that currently have liquid ATOM tokens first need to stake their tokens before they can lock them in Hydro. 
There are different locking durations available, and the longer the duration, the more voting power the user receives. 
Once the locking period is over, the user can unlock their locked tokens and will receive the original amount of tokens back. 
It is also possible to refresh a partially expired lockup to gain more voting power (read more about *Voting* below).
```
Do not use the iqlusion repo as it's probably outdated.

Author: linked forum post

For security reasons, there is also a maximal number of LSM shares that can be locked in the contract. This number can be updated by the whitelist admin (see more in the section on the "Permission structure" below).

## Tranches, Rounds, Proposals

> The Hydro contract supports multiple tranches. In practice, there will be a general tranche accessible to all projects, and a tranche for projects using the ICS platform.

Reviewer: ```suggestion
The Hydro contract supports multiple tranches. In practice, there will be a general tranche accessible to all projects, and a tranche for projects using [Interchain Security (ICS)](https://cosmos.github.io/interchain-security/).
```

Namely, an LSM share represents one share of a validator, and thus a claim to a part of the underlying Atom of that validator.

> However, one LSM share does *not* represent one staked Atom directly. Instead, different validators have different ratios of share<>staked Atom (we call this the "power ratio"). This means that the Hydro contract needs to keep track of the power ratios of all validators that users can lock tokens from. Additionally, in the worst case we need to iterate over all the different validators that users hold tokens from to calculate the voting power of a user and the scores of proposals.

Reviewer: ```suggestion
However, one LSM share does *not* represent one staked ATOM directly. 
Instead, different validators have different ratios of share<>staked ATOM (we call this the "power ratio"). 
This means that the Hydro contract needs to keep track of the power ratios of all validators that users can lock tokens from. 
Additionally, in the worst case we need to iterate over all the different validators that users hold tokens from to calculate the voting power of a user and the scores of proposals.
```
